py_src/ipyelk/nx/transformer.py

```diff
@@ -389,6 +463,20 @@ def closest_common_visible(self, nodes: Tuple[Hashable]) -> Hashable:
         result = lowest_common_ancestor(tree, [self.closest_visible(n) for n in nodes])
         return result
 
+    def from_id(self, element_id: str) -> Hashable:
+        """Use the elk identifiers to find original objects"""
+        try:
+            return self._elk_to_item[element_id]
+        except KeyError:
+            raise ValueError(f"Element id `{element_id}` not in elk id registry.")
```

Comment:
perhaps make some `ElkError` subclass... built-in exception types are harder to reason about.